### ISSUE TITLE
Add an option to perform a dry run of a release

### ISSUE THREAD

**Issue by mttkay** [researchgate/gradle-release#114]
Like maven-release-plugin does. It's really painful to work with release scripts that don't support dry runs if you add any form of customization/additional steps.

**Comment by danez** [comment]
Good idea, and totally possible for all the things the plugin does on its own, but what I'm just worried about is when people hook up there stuff (especially publish and upload) which would still be triggered.

**Comment by pviniciusfm** [comment]
UP

**Comment by samuel-beniamin** [comment]
Is there any updates on this issue ?

**Comment by damnhandy** [comment]
@danez in that case, why not make a `dryRun` option a separate task rather than a flag on the existing release task? Reusing the existing `release` task would like be ripe for user error. But if it were a separate task, it would require someone to do some work in order to do the wrong thing.

**Comment by rupebac** [comment]
+1 to this

**Comment by gvonfang** [comment]
bump on this thread.

**Comment by Tholian** [comment]
I would love to see this possibility as well.

**Comment by curiousjazz77** [comment]
+ 1
